lookout/style/typos/analyzer.py

```diff
@@ -107,6 +107,57 @@ def analyze(self, ptr_from: ReferencePointer, ptr_to: ReferencePointer,
                         comments.append(comment)
         return comments
 
+    @staticmethod
+    def reconstruct_identifier(tokenizer: TokenParser, pred_tokens: List[str], identifier: str) \
+            -> str:
+        """
+        Reconstruct identifier given predicted tokens and initial identifier.
+
+        :param tokenizer: tokenizer - instance of TokenParser.
+        :param pred_tokens: list of predicted tokens.
+        :param identifier: identifier.
+        :return: reconstructed identifier based on predicted tokens.
+        """
+        identifier_l = identifier.lower()
+        # setup required parameters
+        single_shot = tokenizer._single_shot
+        min_split_length = tokenizer._min_split_length
```

Comment:
@irinakhismatullina What is the current minimum analyzed identifier length?

Reply:
There are no lower bounds for now

Reply:
I suggest to not fix typos in parts less than 3 characters